dashboard/src/pages/TreeDetails/Tabs/Tests/TestsTab.tsx

```diff
@@ -106,6 +108,16 @@ const TestsTab = ({ reqFilter }: TestsTabProps): JSX.Element => {
     );
   }
 
+  const failedCount = groupStatus({
+    doneCount: data.testStatusSummary.DONE ?? 0,
+    errorCount: data.testStatusSummary.ERROR ?? 0,
+    failCount: data.testStatusSummary.FAIL ?? 0,
```

Comment:
You can get the faildCount directly of `data.testStatusSummary.FAIL` without use the `groupStatus` function

Reply:
That's true. After your comment I read the groupStatus function more carefully and figured I didn't need this in this case to get the fail count! However the new version of the code don't use this any more.